src/nbl/CMakeLists.txt

```diff
@@ -582,19 +559,25 @@ if (UNIX AND NOT ANDROID AND NOT APPLE)
 		${X11_xf86vmode_INCLUDE_PATH})
 endif()
 
-
 if (NBL_COMPILE_WITH_CUDA)
 	target_include_directories(Nabla PUBLIC ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
 endif()
 
 target_compile_definitions(Nabla 
-	PRIVATE	_NBL_STATIC_LIB_ EGL_STATIC SPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS $<$<BOOL:${WIN32}>:VK_USE_PLATFORM_WIN32_KHR>
-	PUBLIC	UNW_LOCAL_ONLY
+	PRIVATE	_NBL_STATIC_LIB_ EGL_STATIC SPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS
+	PUBLIC UNW_LOCAL_ONLY
 )
+
+if(WIN32)
+	target_compile_definitions(Nabla PRIVATE VK_USE_PLATFORM_WIN32_KHR)
+endif()
+
 if (ANDROID)
 	target_compile_definitions(Nabla PUBLIC $<$<BOOL:${OpenGL_EGL_FOUND}>:WL_EGL_PLATFORM>)
```

Comment:
isnt `WL_EGL_PLATFORM` wayland?

btw nothing should depend on `*_FOUND` in the OpenGL/EGL/GLX/Vulkan space, this is why we dynamically link stuff so even if the dev machine doesn't have something we're able to compile for everything

Reply:
ask @Crisspl please, but the `VK_USE_PLATFORM_WIN32_KHR` change is due to generating definitions to the header file.